### ISSUE TITLE
Dependencies for the classroom

### ISSUE THREAD

**Issue by dpshelio** [UCL/2022-09-19-UCL-DaSH#2]
If there are extra dependencies, we will create a setup page.
`setup/index.md`
@tdveale

**Comment by tdveale** [comment]
Extra dependencies for IDEAS day:

Bash
- dcm2niix

Python
- nibabel
- nilearn
- statsmodels

Anymore from your notebooks @mary-tziraki ?

**Comment by dpshelio** [comment]
Does `dcm2niix` works on git for windows? Or would they need to run it from powershell? Can we avoid it completely?

**Comment by mary-tziraki** [comment]
This works even at the Jupiter notebook and we intent to use the notebooks 
We cannot avoid it because we change the image format from DICOm to NIfTI and then we will use the NiBabel library to handle the images in NIfTI format

**Comment by dpshelio** [comment]
It could be avoided if we provide the images in the right format from the start.
In any case, it seems they [provide a conda package](https://github.com/rordenlab/dcm2niix#install). Still, we need to test whether that package can be installed and run on a windows machine. I won't be able to test it till I'm back, so I suggest someone does it and find out whether it can be executed from the git bash terminal. Please, write the steps to get it working there in this issue, we will then add them to the setup instructions. It would be too difficult to change the material closer to the workshop date.
If you don't have a windows machine, you can install it on a virtual machine using an image from [UCL](https://swdb.ucl.ac.uk/) and follow the steps to install git bash and anaconda as suggested in our workshop page.

**Comment by tdveale** [comment]
I installed git for windows and anaconda on my Windows 10 machine using the [setup instructions](http://github-pages.ucl.ac.uk/2022-09-19-UCL-DaSH/) and tested `dcm2niix`. It works and I was able to run the command and relevant cells in the jupyter notebook that uses it. 

You're right @dpshelio , the conda install seems to be the way to go. Steps are below:
1. Ensure git for windows and anaconda are installed [instructions here](http://github-pages.ucl.ac.uk/2022-09-19-UCL-DaSH/)
2. Open the gitbash terminal 
3. Type `conda install -c conda-forge dcm2niix` and press enter
4. After the install is complete, type `dcm2niix` and press enter
5. If a message describing options for the command appears, the installation has been successful!

Doing the DICOM to NIfTI conversion is an important first step in neuroimaging, so it would be helpful for learners to get hands on experience with it.
